kafka-rest/src/main/java/io/confluent/kafkarest/extension/ResourceBlocklistFeature.java

```diff
@@ -88,14 +92,15 @@ public void configure(ResourceInfo resourceInfo, FeatureContext context) {
     String value();
   }
 
+  @Priority(Priorities.AUTHENTICATION - 1)
```

Comment:
Can you create a class `io.confluent.kafkarest.extension.MorePriorities` (to avoid clashing with `javax.ws.rs.Priorities`), and define a constant PRE_AUTHENTICATION (possibly equal to AUTHENTICATION - 500)?